ang_vel: 0.2       #Angular velocity of husky for circular trajectory
linear_vel: 2    #Linear velocity of husky for circular trajectory    

